landscaper/pkg/controlplane/blueprint/export-execution.yaml

```diff
@@ -0,0 +1,17 @@
+exports:
+  gardenerAPIServerCA: '{{- index .values "deployitems" "deploy" "gardenerAPIServerCA"
+    | toYaml | nindent 4 }}'
```

Comment:
Does this actually work when the `| toYaml | nindent 4` part is on a new line?

Reply:
Yep, so far when I tested it there were no problems as it is merely a string.